casper/src/test/scala/coop/rchain/casper/helper/TestNode.scala

```diff
@@ -506,6 +454,26 @@ object TestNode {
                          RuntimeManager(rSpaceStore, mStore, Genesis.NonNegativeMergeableTagName)
                        )
 
+      shardConf = CasperShardConf(
+        faultToleranceThreshold = 0,
+        shardName = genesis.shardId,
+        finalizationRate = 1,
+        maxNumberOfParents = maxNumberOfParents,
+        maxParentDepth = maxParentDepth.getOrElse(Int.MaxValue),
+        synchronyConstraintThreshold = synchronyConstraintThreshold.toFloat,
+        heightConstraintThreshold = Long.MaxValue,
+        // Validators will try to put deploy in a block only for next `deployLifespan` blocks.
+        // Required to enable protection from re-submitting duplicate deploys
+        deployLifespan = 50,
+        casperVersion = 1,
+        configVersion = 1,
+        bondMinimum = 0,
+        bondMaximum = Long.MaxValue,
+        epochLength = 10000,
+        quarantineLength = 20000,
+        minPhloPrice = 1
+      )
```

Comment:
For now shard config is created inside TestNode, it will probably be changed in subsequent PRs.